src/server/auth/google-auth.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const https = require('https')
+const queryString = require('querystring')
+
+// Generating the URL for the Google auth.
+// This is a promise in case we need to generate a random token later on.
+exports.start = (redirectURI, clientID) => {
+  return new Promise((resolve, reject) => {
+    const authURL = [
+      'https://',
+      'accounts.google.com',
+      '/o/oauth2/v2/auth?',
+      queryString.stringify({
+        'client_id': clientID,
+        'redirect_uri': redirectURI,
+        'response_type': 'code',
+        'scope': 'profile email openid'
+      })].join('')
+    resolve(authURL)
```

Comment:
You can remove the var authURL here (by putting the calls to build that variable into the call to resolve).